pkg/util/trace.go

```diff
@@ -0,0 +1,56 @@
+package util
+
+import (
+	"context"
+
+	oc "go.opencensus.io/trace"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/propagation"
+	otel "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/codes"
+	"google.golang.org/grpc/status"
+)
+
+// PropagateOpenTelemetryToOpenCensus arranges for spans that let us trace into libraries using OpenCensus.
+func PropagateOpenTelemetryToOpenCensus(ctx context.Context) context.Context {
```

Comment:
I thought I'd help you out with this PR by removing CloudBlobAccess from the tree.

This should allow you to trim down this PR by removing any OpenCensus related bits, right?

Reply:
Rebased and cleaned up.